meta-lmp-base/recipes-bsp/u-boot/u-boot-ostree-scr-fit/boot-common.cmd.in

```diff
@@ -62,6 +62,8 @@ if test "${fiovb.is_secondary_boot}" = "0"; then
 		if test ! $? -eq 0; then fiovb write_pvalue upgrade_available 0; fi
 		fiovb read_pvalue bootupgrade_available 4
 		if test ! $? -eq 0; then fiovb write_pvalue bootupgrade_available 0; fi
+		fiovb read_pvalue bootfirmware_version 4
+		if test ! $? -eq 0; then run bootcmd_bootenv; fiovb write_pvalue bootfirmware_version ${bootfirmware_version}; fi
```

Comment:
`read_pvalue <name> <bytes> - read a persistent value <name>`

I think bootfirmware_version can be a lot larger than 4 bytes (e.g. when using md5).

https://ci.foundries.io/projects/lmp/builds/1294/imx8mmevk/artifacts/other/lmp-boot-firmware/version.txt

`bootfirmware_version=9a6c02d769df5f05f6ea924b9666af91-b7130a8ab02c02e8a6df7d4beaaad2c1`

Reply:
very true, how big should it be? 40 bytes 80?

Reply:
that string is 86 I will set it to 90?

Reply:
I would probably use something like 128, to be safe.